models/comment.rb

```diff
@@ -108,6 +108,7 @@ def to_hash(params={})
                  .merge("votes" => votes.slice(*%w[count up_count down_count point]))
                  .merge("abuse_flaggers" => abuse_flaggers)
                  .merge("type" => "comment")
+                 .merge("child_comments_count" => Comment.where({"parent_id" => _id}).count())
```

Comment:
I haven't worked with the `cs_comments_service` before so I only have two questions:
1. Is `child_comments_count` the count of the `children` of the comment? If yes, wouldn't `children_count` be a more appropriate name?
2. Are we adding an additional query to each load of a comment?